tests/test_micropip.py

```diff
@@ -200,12 +200,25 @@ def selenium_standalone_micropip(selenium_standalone):
     """Import micropip before entering test so that global initialization of
     micropip doesn't count towards hiwire refcount.
     """
-    selenium_standalone.run_js(
-        """
-        await pyodide.loadPackage("micropip");
-        pyodide.runPython("import micropip");
-        """
-    )
+
+    wheel_dir = Path(__file__).parent.parent / "wheel"
+    wheel_files = list(wheel_dir.glob("*.whl"))
+
+    if not wheel_files:
+        pytest.exit("No wheel files found in wheel/ directory")
```

Comment:
Good point, thanks!

Should we just create a wheel at runtime here (maybe using their [programmatic API](https://pypa-build.readthedocs.io/en/stable/api.html#build.ProjectBuilder)? Otherwise, it would be annoying to use these tests locally if one has to always re-generate wheels before running tests.

Reply:
> Should we just create a wheel at runtime here (maybe using their [programmatic API](https://pypa-build.readthedocs.io/en/stable/api.html#build.ProjectBuilder)? Otherwise, it would be annoying to use these tests locally if one has to always re-generate wheels before running tests.

Thanks for the info! I'll try to use that API.

Reply:
Okay, I managed to make it work.